pkg/server/server.go

```diff
@@ -124,12 +131,26 @@ func WithListObjectsMaxResults(limit uint32) OpenFGAServiceV1Option {
 	}
 }
 
+// WithMaxConcurrentReadsForListObjects sets a limit on the number of datastore reads that can be in flight for a given Check call.
+// This number should be set depending on the RPS expected for Check and ListObjects APIs, the number of OpenFGA replicas running,
```

Comment:
This is for ListObjects, not also Check.

Reply:
Right, but it's important to account for traffic for both when setting these values